easybuild/easyconfigs/m/meRanTK/meRanTK-1.1.1b.eb

```diff
@@ -3,35 +3,27 @@
 # Homepage: https://www.adelaide.edu.au/phoenix/
 #
 # Copyright:: 	adelaide.edu.au/phoenix  
-# Authors::	Robert Qiao <robert.qiao@adelaide.edu.au>, Exequiel Manuel Sepulveda Escobedo <exequiel.sepulvedaescobedo@adelaide.edu.au> 
+# Authors::	Robert Qiao <robert.qiao@adelaide.edu.au>, Exe Escobedo <exequiel.sepulvedaescobedo@adelaide.edu.au>
 # License::
 #
 # Notes::
 ##
 
+easyblock = 'BinariesTarball'
+
 name = 'meRanTK'
 version = '1.1.1b'
 
-easyblock = 'BinariesTarball'
 homepage = 'http://www.icbi.at/index.html'
 description = """meRanTK is a versatile high performance toolkit for complete analysis of methylated RNA data."""
 
 toolchain = {'name': 'dummy', 'version': 'dummy'}
 
-source_urls = ['http://www.icbi.at/software/meRanTK/downloads']
-sources = ['%s/meRanTK-%s.zip' %(version,version,)]
-
-dependencies = [
-]
+source_urls = ['http://www.icbi.at/software/meRanTK/downloads/%(version)s/']
+sources = ['meRanTK-%(version)s.zip']
 
 sanity_check_paths = {
-	'files':['bin/meRanAnnotate'],
-	'files':['bin/meRanCall'],
-	'files':['bin/meRanCompare'],
-	'files':['bin/meRanGs'],
-	'files':['bin/meRanGt'],
-	'files':['bin/meRanT'],
-	'dirs':[],
```

Comment:
`'dirs': [],` is still required (that's why Travis is failing), sorry if I wasn't clear